OsmAnd-java/src/main/java/net/osmand/gpx/GPXUtilities.java

```diff
@@ -1753,6 +1753,19 @@ public static GPXFile loadGPXFile(InputStream stream, GPXExtensionsReader extens
 						Object pop = parserState.pop();
 						assert pop instanceof WptPt;
 					}
+				} else if (tok == XmlPullParser.TEXT) {
+					GPXExtensions parse = parserState.peek();
+					if (parse instanceof Author) {
+						String text = parser.getText();
+						if(((Author) parse).name == null){
+							((Author) parse).name = text;
+						}
+					} else if (parse instanceof Copyright) {
+						String text = parser.getText();
+						if (((Copyright) parse).author == null) {
+							((Copyright) parse).author = text;
+						}
+					}
```

Comment:
Looks wrong. Does not conform to https://www.topografix.com/GPX/1/1/#type_metadataType